core/trino-main/src/test/java/io/trino/sql/planner/TestTableFunctionInvocation.java

```diff
@@ -238,31 +238,35 @@ public void testRemoveRedundantTableFunction()
         assertPlan("SELECT * FROM TABLE(mock.system.pass_through_function(input => TABLE(SELECT 1, true WHERE false) t(a, b) PRUNE WHEN EMPTY))",
                 output(values(ImmutableList.of("x", "a", "b"))));
 
-        assertPlan("""
+        assertPlan(
+                        """
```

Comment:
Indentation seems wrong here and in the ones below, too.

Reply:
Yeah - but to be honest I am not sure how should it be. 
Also no matter what I do automatic reformat in intellij breaks it anyway to:

```
        assertPlan(
                """
                        SELECT *
                        FROM TABLE(mock.system.two_table_arguments_function(
                                        input1 => TABLE(SELECT 1, true WHERE false) t1(a, b) PRUNE WHEN EMPTY,
                                        input2 => TABLE(SELECT 2, false) t2(c, d) KEEP WHEN EMPTY))
                        """,
```


This PR brings back the discussion about automatic code reformatting. IMO it would be great to have. One less thing to think of - even if not optimal from time to time.

Reply:
See https://github.com/trinodb/trino/pull/21815#discussion_r1590009280

Reply:
Yeah - this is a mess.
I really think we should commit on using Intellij formatter and plug it in as a headless verifier in CI pipeline to verify if checked in code matches the formatting